tls_codec/src/quic_vec.rs

```diff
@@ -311,8 +302,16 @@ impl VLBytes {
         &self.vec
     }
 
-    fn vec_mut(&mut self) -> &mut Vec<u8> {
-        &mut self.vec
+    /// Add an element to this.
+    #[inline]
+    fn push(&mut self, v: u8) {
```

Comment:
The compiler says this is dead code. Also, isn't this the function that you deleted from the macro above? I'm not sure I understand this change.